packages/Makefile

```diff
@@ -7,6 +7,8 @@ all: pyodide-build
 	mkdir -p $(HOSTINSTALLDIR) $(WASM_LIBRARY_DIR)
 	PYODIDE_ROOT=$(PYODIDE_ROOT) pyodide build-recipes \
 	"$(PYODIDE_PACKAGES)" \
+	--install \
+	--install-dir=../dist \
```

Comment:
Would it make sense to combine these into one flag? Or is there a default `install-dir`?

Reply:
>  Or is there a default install-dir?

I set default install dir to `$(pwd)/dist` (not PYODIDE_ROOT/dist because I want it to make it work out-of-tree someday).

> Would it make sense to combine these into one flag?

I didn't want to mix boolean and string flag (and I guess it is pretty hard to do so with typer).